packages/npm-dependencies-validation/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@sap-devx/npm-dependencies-validation",
+  "version": "1.3.0",
+  "private": true,
+  "description": "",
+  "author": "SAP SE",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/app-studio-toolkit"
+  },
+  "main": "./dist/src/index",
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "test": "mocha",
+    "compile:watch": "tsc -p ./ -watch",
```

Comment:
the `watch` script may be redundant as we use `compile:watch` from the root of the repo
which compiles **all** the projects.